### PR TITLE
fix syncbn, test=develop

### PR DESCRIPTION
**Pull request by ceci3**
### PR types

Others

### PR changes

APIs

### Describe

use_global_stats  only  support use_global_stats = false, fix it
move syncbn from fluid/dygraph to paddle/nn/norm
pre PR: https://github.com/PaddlePaddle/Paddle/pull/26032

### CODE REVIEW

File: python/paddle/nn/layer/norm.py [hunk 1]
```diff
@@ -14,15 +14,239 @@
...
+        self._weight_attr = weight_attr
+        self._bias_attr = bias_attr
+        self._num_features = num_features
+        self._data_layout = data_format
+        self._momentum = momentum
+        self._epsilon = epsilon
+        self._track_running_stats = track_running_stats
+
+        if self._track_running_stats == False:
+            logging.warn(
```
Comment (willthefrog) [inline python/paddle/nn/layer/norm.py]: use `warnings` instead of `logging`

**Review by willthefrog — CHANGES_REQUESTED** [review]

File: python/paddle/nn/layer/norm.py (hunk 1 above)
Comment (ceci3) [reply python/paddle/nn/layer/norm.py]: done, thanks

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by jzhang533 — APPROVED** [review]
lgtm

**Review by willthefrog — APPROVED** [review]
LGTM

**Pull request merged**